src/components/learner-credit-management/BudgetDetailPageOverviewAvailability.jsx

```diff
@@ -94,6 +94,46 @@ const BudgetActions = ({
     );
   }
 
+  if (status === BUDGET_STATUSES.retired) {
```

Comment:
[nit/suggestion] Similar suggestion as above. Could be worth splitting out into its own component.

Reply:
Since this ticket is basically in a complete state, you can go ahead and make a new ticket to tackle that bit of work. Its important to make the code as readable as possible, and the long list of return statements with JSX components as each return makes this file difficult to parse. 
You can also add a // TODO: comment as a reminder to handle that optimization in the future. 👍🏽